### PR TITLE
Move the required parent param to AcquireTokenInteractive.WithParentW…

### PR DESCRIPTION
**Pull request by bgavrilMS**
…indowOrActivity

### CODE REVIEW

**Comment by jmprieur** [comment]
Impact in term of DevEx:

**Samples to fix:** (all in their Msal3x branch)
- https://github.com/azure-samples/active-directory-dotnet-desktop-msgraph-v2
- https://github.com/azure-samples/active-directory-dotnet-native-uwp-v2
- https://github.com/azure-samples/active-directory-xamarin-native-v2

**Quickstarts to update** [Needs to happen **quickly** because impacts in-portal experience]
- https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-v2-uwp
- https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-v2-windows-desktop

**Tutorials to update**
- https://docs.microsoft.com/en-us/azure/active-directory/develop/tutorial-v2-windows-uwp
- https://docs.microsoft.com/en-us/azure/active-directory/develop/tutorial-v2-windows-desktop

File: src/Microsoft.Identity.Client/ApiConfig/AcquireTokenInteractiveParameterBuilder.cs [hunk 1]
```diff
@@ -156,53 +157,172 @@ public AcquireTokenInteractiveParameterBuilder WithPrompt(Prompt prompt)
...
+        #region WithParentActivityOrWindow
+
+        /*
+         * .WithParentActivityOrWindow is platform specific but we need a solution for
+         * projects like XForms where code is shared from a netstandard assembly. So expose
+         * a variant of .WithParentActivityOrWindow that allows users to inject the parent as an object,
+         * since Activity, ViewController etc. do not exist in NetStandard.
+         */
+
+#if RUNTIME || NETSTADARD_BUILDTIME
```
Comment (jennyf19) [inline src/Microsoft.Identity.Client/ApiConfig/AcquireTokenInteractiveParameterBuilder.cs]: >ADARD [](start = 20, length = 5)

sp.

Comment (bgavrilMS) [reply src/Microsoft.Identity.Client/ApiConfig/AcquireTokenInteractiveParameterBuilder.cs]: Thx, great catch!

**Comment by bgavrilMS** [comment]
Passing build: https://identitydivision.visualstudio.com/IDDP/_build/results?buildId=324046

**Comment by bgavrilMS** [comment]
Merged separately.

**Pull request closed without merging**